### PR TITLE
Subthemes in print templates

### PR DESCRIPTION
**Pull request by pfirpfel**
If `split_sub_themes` is enabled, each subtheme will get a separate page in the pdf output.
Per default, this behaviour is disabled and should not affect anyone even if they have values in the subtheme field.

### CODE REVIEW

**Comment by jwkaltz** [comment]
Addresses issue #372

**Comment by peterschaer** [comment]
Looks good on our data:

[20181121040025_extract.pdf](https://github.com/camptocamp/pyramid_oereb/files/2604710/20181121040025_extract.pdf)
[20181121040245_extract.pdf](https://github.com/camptocamp/pyramid_oereb/files/2604711/20181121040245_extract.pdf)

Of course in the french extract the subtopic is in german. But this will be adressed in #705 

the order of the subtopics is somewhat random although there is an order defined in the weisung. I'll check if reordering the items in the database will help with that.

**Comment by jwkaltz** [comment]
@vvmruder FYI

File: print/print-apps/oereb/config.yaml [hunk 1]
```diff
@@ -106,6 +106,8 @@ templates:
                         default: ''
                     SubTheme: !string
```
[the remark this answers is not in the document]
Comment (pfirpfel) [reply print/print-apps/oereb/config.yaml]: Possible improvement:
https://github.com/camptocamp/pyramid_oereb/compare/print-subthemes...pfirpfel:print-subthemes

Replaces the slightly confusing SubTheme_Text with a boolean flag Split_SubTheme. This variant uses the already existing SubTheme field. The PDF output is exactly the same.

[the remark this answers is not in the document]
Comment (vvmruder) [reply print/print-apps/oereb/config.yaml]: +1

**Pull request merged**